DESCRIPTION

```diff
@@ -1,13 +1,13 @@
 Package: dash
 Title: An Interface to the Dash Ecosystem for Authoring Reactive Web Applications
-Version: 0.2.0
+Version: 0.3.0
 Authors@R: c(person("Chris", "Parmer", role = c("aut"), email = "chris@plot.ly"), person("Ryan Patrick", "Kyle", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5829-9867"), email = "ryan@plot.ly"), person("Carson", "Sievert", role = c("aut"), comment = c(ORCID = "0000-0002-4958-2844")), person(family = "Plotly Technologies", role = "cph")) 
 Description: A framework for building analytical web applications, Dash offers a pleasant and productive development experience. No JavaScript required.
 Depends:
   R (>= 3.0.2)
 Imports:
   dashHtmlComponents (== 1.0.2),
-  dashCoreComponents (== 1.6.0),
+  dashCoreComponents (== 1.7.0),
```

Comment:
```suggestion
  dashCoreComponents (>= 1.7.0),
```
Probably best to change this to greater than, as 1.8.0 of dash-core-components has been released to master.